src/debug.ts

```diff
@@ -157,6 +214,32 @@ export async function debugTests(
   await deferredPromise.promise
 }
 
+function getTestProjectsInFolder(path: string, api: VitestFolderAPI, tree: TestTree) {
+  const folder = tree.getOrCreateFolderTestItem(api, path)
+  const items = tree.getFolderFiles(folder)
+  return items.map(item => (getTestData(item) as TestFile).project)
+}
+
+function getProjectsFromTests(item: vscode.TestItem | undefined, api: VitestFolderAPI, tree: TestTree): string[] {
+  const items = getTestProjectsInFolder(item?.uri?.fsPath ?? '', api, tree)
+  if (items.length > 0) {
+    return items
+  }
+  // Climb up tree until entry with project is found
+  if (item?.parent) {
+    return getProjectsFromTests(item.parent, api, tree)
+  }
+  return []
+}
+
+function getBrowserModeLaunchArgs(isPlaywright: boolean, config: any): string {
+  const browser = isPlaywright ? 'chromium' : 'chrome'
```

Comment:
I like that we provide the browser configuration ourselves. I think at one point there should be a UI in the extension to select the browser, but it's not related to these changes